### PR TITLE
Adam12 issue 119/blog tutorial improvements

### PR DESCRIPTION
**Pull request by wikimatze**
Take the PR (https://github.com/padrino/padrino-docs/pull/120) of @adam12 as a foundation. Have switched from AR to Sequel, remove deployment section.

Do you think we should also remove the old  Padrino 0.12.2 part as well as the sceencast?

### CODE REVIEW

File: guides/02_getting-started/04_blog-tutorial.html.md [hunk 1]
```diff
@@ -257,29 +259,35 @@ the `-a` option to the command - this is handy if you would like to have models
 should be coped only to sub-apps.
 
 ```shell
-$ padrino g model post title:string body:text
-       apply  orms/activerecord
+$ padrino g model post title:string body:text created_ad:datetime
```
Comment (adam12) [inline guides/02_getting-started/04_blog-tutorial.html.md]: This should likely be `created_at`.

File: guides/02_getting-started/04_blog-tutorial.html.md [hunk 2]
```diff
@@ -410,64 +433,99 @@ $ padrino g migration AddAccountToPost account_id:integer
...
-class AddAccountToPost < ActiveRecord::Migration
-  def self.up
-    change_table :posts do |t|
-      t.integer :account_id
+
+Sequel.migration do
+  class Account < Sequel::Model; end
+  class Post < Sequel::Model
+    many_to_one :account
+  end
```
Comment (adam12) [inline guides/02_getting-started/04_blog-tutorial.html.md]: I don't think we need these class definitions in this migration file.

File: guides/02_getting-started/04_blog-tutorial.html.md [hunk 3]
```diff
@@ -410,64 +433,99 @@ $ padrino g migration AddAccountToPost account_id:integer
...
+
+And change the content of the migration:
+
+```ruby
+# db/migrate/004_migrate_existing_posts_to_first_account.rb
+Sequel.migration do
+  class Account < Sequel::Model; end
+  class Post < Sequel::Model
+    many_to_one :account
+  end
```
Comment (adam12) [inline guides/02_getting-started/04_blog-tutorial.html.md]: Let's drop the class definitions from this migration too. I can't see them being required.

File: guides/02_getting-started/04_blog-tutorial.html.md [hunk 4]
```diff
@@ -31,27 +31,32 @@ mentioned there may not be valid.
...
+
 or even execute the
 [blog tutorial project template](https://github.com/padrino/padrino-recipes/blob/master/templates/sampleblog_template.rb "blog tutorial
 project template") which will automatically build the blog project step by step
 using our excellent template runner. You can do this simply by invoking:
 
+
 ```shell
-$ padrino g project sample_blog --template sampleblog
+$ padrino g project blog-tutorial --template blogtutorial
```
Comment (adam12) [inline guides/02_getting-started/04_blog-tutorial.html.md]: This line won't work as the `blogtutorial` template isn't available.

File: guides/02_getting-started/04_blog-tutorial.html.md [hunk 5]
```diff
@@ -410,64 +433,99 @@ $ padrino g migration AddAccountToPost account_id:integer
...
-Before running the migration we need to add the `models` folder to the `dependency\_paths`:
+And add the association to the Account model:
+
 
 ```ruby
-# config/boots.rb
-Padrino.before_load do
-  Padrino.dependency_paths << Padrino.root('models/**/*.rb')
+class Account < Sequel::Model
+  one_to_one :post
```
Comment (adam12) [inline guides/02_getting-started/04_blog-tutorial.html.md]: This should be

```ruby
one_to_many :posts
```

**Comment by adam12** [comment]
Everything looks good. I ran through it locally and had no issues. :+1: Thanks for carrying this over the finish line.

> Do you think we should also remove the old Padrino 0.12.2 part as well as the sceencast?

I think so. No point in encouraging people to use an extremely old version.

File: guides/02_getting-started/04_blog-tutorial.html.md (hunk 2 above)
Comment (wikimatze) [inline guides/02_getting-started/04_blog-tutorial.html.md]: Agree!

File: guides/02_getting-started/04_blog-tutorial.html.md (hunk 3 above)
Comment (wikimatze) [inline guides/02_getting-started/04_blog-tutorial.html.md]: Okay

File: guides/02_getting-started/04_blog-tutorial.html.md (hunk 4 above)
Comment (wikimatze) [inline guides/02_getting-started/04_blog-tutorial.html.md]: I have to create this.

File: guides/02_getting-started/04_blog-tutorial.html.md (hunk 1 above)
Comment (wikimatze) [inline guides/02_getting-started/04_blog-tutorial.html.md]: That's true

File: guides/02_getting-started/04_blog-tutorial.html.md (hunk 4 above)
Comment (wikimatze) [inline guides/02_getting-started/04_blog-tutorial.html.md]: But until this is created, I'll comment this out.

File: guides/02_getting-started/04_blog-tutorial.html.md (hunk 5 above)
Comment (wikimatze) [inline guides/02_getting-started/04_blog-tutorial.html.md]: True

File: guides/02_getting-started/04_blog-tutorial.html.md (hunk 4 above)
Comment (wikimatze) [inline guides/02_getting-started/04_blog-tutorial.html.md]: But until this is created, I'll comment this out.

**Comment by wikimatze** [comment]
I think, it' ready now.

**Comment by adam12** [comment]
:+1: Looks good.

**Pull request merged**